Failing tests with python 3.10 due to changed error message

This is showing up in https://github.com/conda-forge/tokenizers-feedstock/pull/40, almost certainly because the text of the error being raised changed with python 3.10:
```
____________________ TestCustomNormalizer.test_instantiate _____________________

self = <tests.bindings.test_normalizers.TestCustomNormalizer object at 0x7fd354094ee0>

    def test_instantiate(self):
        bad = Normalizer.custom(TestCustomNormalizer.BadCustomNormalizer())
        good_custom = TestCustomNormalizer.GoodCustomNormalizer()
        good = Normalizer.custom(good_custom)
    
        assert isinstance(bad, Normalizer)
        assert isinstance(good, Normalizer)
        with pytest.raises(Exception, match="TypeError: normalize()"):
>           bad.normalize_str("Hey there!")
E           Exception: TypeError: TestCustomNormalizer.BadCustomNormalizer.normalize() missing 1 required positional argument: 'wrong'
```

The good news is that these seem to be the only relevant errors:
```
=========================== short test summary info ============================
FAILED tests/bindings/test_normalizers.py::TestCustomNormalizer::test_instantiate
FAILED tests/bindings/test_pre_tokenizers.py::TestCustomPreTokenizer::test_instantiate
================== 2 failed, 132 passed, 1 skipped in 57.25s ===================
```

Well, aside from the fact that windows does not support forking, so [this line](https://github.com/huggingface/tokenizers/blob/python-v0.11.6/bindings/python/tests/utils.py#L102) should be guarded to not run on windows.

--- a/bindings/python/tests/bindings/test_normalizers.py
+++ b/bindings/python/tests/bindings/test_normalizers.py
@@ -148,7 +148,7 @@ class TestCustomNormalizer:
 
         assert isinstance(bad, Normalizer)
         assert isinstance(good, Normalizer)
-        with pytest.raises(Exception, match="TypeError: normalize()"):
+        with pytest.raises(Exception, match="TypeError:.*normalize()"):
             bad.normalize_str("Hey there!")
         assert good.normalize_str("Hey there!") == "Hey you!"
         with pytest.raises(

--- a/bindings/python/tests/bindings/test_pre_tokenizers.py
+++ b/bindings/python/tests/bindings/test_pre_tokenizers.py
@@ -213,7 +213,7 @@ class TestCustomPreTokenizer:
 
         assert isinstance(bad, PreTokenizer)
         assert isinstance(good, PreTokenizer)
-        with pytest.raises(Exception, match="TypeError: pre_tokenize()"):
+        with pytest.raises(Exception, match="TypeError:.*pre_tokenize()"):
             bad.pre_tokenize_str("Hey there!")
         assert good.pre_tokenize_str("Hey there!") == [
             ("Hey there!", (0, 10)),

